### PR TITLE
Logistic Regression notes

### PR DESCRIPTION
**Pull request by jeremy-sanchez**
MCR added into notes; false negatives and false positives added into notes

### CODE REVIEW

File: 6-prediction/05-logistic-regression/notes.qmd [hunk 1]
```diff
@@ -139,57 +155,115 @@ We can visualize the approach that logistic regression takes by sketching the pr
...
+
+### Evaluating model performance
+
+#### Calculating $\text{MCR}$
+
+Let's take a look at the predictions that this model makes back into the data set that it was trained on. When using the `predict()` function on logistic regression models, there are several different types of predictions that it can return, so be sure to use the additional argument `type` and supply it with `"response"`. This will return $\hat{p}_i$ (probability of a `1`) for each observation.
 
 ```{r}
 #| code-fold: false
+p_hat_m1 <- predict(object = m1,
```
Comment (andrewpbray) [inline 6-prediction/05-logistic-regression/notes.qmd:206]: Would it be worth folding this with subsequent code into blocks that look like the block from the last PS? That’s to say: a series of mutates before the summarize?

File: 6-prediction/05-logistic-regression/notes.qmd [hunk 2]
```diff
@@ -139,57 +155,115 @@ We can visualize the approach that logistic regression takes by sketching the pr
...
 ```
 
-We can then move from values of $\hat{p}_i$ to values of $y_1$ by checked to see whether each value of $\hat{p}_i$ is greater than .5. Let's combine these values into one data frame.
+Finally, we can calculate $\text{MCR}$. First, we can find all of the emails for which the `y` and `y_hat_m1` don't match. Then, we can take the *proportion* of those e-mails out of all e-mails in the dataset.
 
 ```{r}
 #| code-fold: false
+email |>
+  mutate(misclassified = (spam != y_hat_m1)) |>
+  summarise(MCR = mean(misclassified))
```
Comment (andrewpbray) [inline 6-prediction/05-logistic-regression/notes.qmd:227]: It’s great how closely this mirrors MSE!

File: 6-prediction/05-logistic-regression/slides.qmd [hunk 3]
```diff
@@ -245,11 +234,11 @@ test %>%
...
 ```{r}
 #| echo: true
-test %>%
-  select(sex) %>%
+test |>
+  select(sex) |>
   mutate(p_hat = p_hat,
-         y_hat = ifelse(p_hat > .5, "male", "female")) %>%
-  summarize(misclas = mean(sex != y_hat))
+         y_hat = ifelse(p_hat > .5, "male", "female")) |>
```
Comment (andrewpbray) [inline 6-prediction/05-logistic-regression/slides.qmd:240]: Whatever you decide on for the mcr code should be mirrored here

**Review by andrewpbray — APPROVED** [review]

File: 6-prediction/05-logistic-regression/notes.qmd (hunk 1 above)
Comment (jeremy-sanchez) [reply 6-prediction/05-logistic-regression/notes.qmd:206]: That's a good idea. I will approve, merge and publish right now, but this is worth adding an issue for.

**Pull request merged**